sdk/eventhubs/azure_messaging_eventhubs/src/common/authorizer.rs

```diff
@@ -611,8 +611,8 @@ mod tests {
         authorizer
             .set_token_refresh_times(TokenRefreshTimes {
                 before_expiration_refresh_time: Duration::seconds(5),
-                jitter_min: -1,
-                jitter_max: 1,
+                jitter_min: Duration::milliseconds(-500),
```

Comment:
Different still from above, and not using the constants. Or, as I'm starting to wonder, is this intentional?

Reply:
100% intentional - jitter intentionally introduces noise in the system, but noise is horrible for deterministic tests.